command/credits.go

```diff
@@ -0,0 +1,248 @@
+package command
+
+import (
+	"bytes"
+	"fmt"
+	"math"
+	"math/rand"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/cli/cli/utils"
+)
+
+var isWindows bool
+
+var thankYou = `
+     _                    _
+    | |                  | |
+_|_ | |     __,   _  _   | |           __
+ |  |/ \   /  |  / |/ |  |/_)   |   | /  \_|   |
+ |_/|   |_/\_/|_/  |  |_/| \_/   \_/|/\__/  \_/|_/
+                                   /|
+                                   \|
+                              _
+                           o | |                           |
+ __   __   _  _  _|_  ,_     | |        _|_  __   ,_    ,  |
+/    /  \_/ |/ |  |  /  |  | |/ \_|   |  |  /  \_/  |  / \_|
+\___/\__/   |  |_/|_/   |_/|_/\_/  \_/|_/|_/\__/    |_/ \/ o
+
+
+`
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+	isWindows = runtime.GOOS == "windows"
+
+	RootCmd.AddCommand(creditsCmd)
```

Comment:
At first I was thinking that maybe this command belongs to under `gh repo`, e.g. `gh repo credits [<repo>]`. Since this command basically displays information about a given repository, grouping it together with other `gh repo` commands made sense to me.

However, since the default is showing credits for `cli/cli` (as opposed to using the current repo), for that reason alone I think it deserves to stay as top-level command.

Reply:
that was my thinking as well (about how it's mainly for seeing cli/cli's credits)